deployment/terraform/create.go

```diff
@@ -994,9 +994,6 @@ func (t *Terraform) updateAppConfig(siteURL string, sshc *ssh.Client, jobServerE
 	cfg.SqlSettings.MaxIdleConns = model.NewPointer(100)
 	cfg.SqlSettings.MaxOpenConns = model.NewPointer(100)
 	cfg.SqlSettings.Trace = model.NewPointer(false) // Can be enabled for specific tests, but defaulting to false to declutter logs
-	if t.output.HasElasticSearch() {
-		cfg.SqlSettings.DisableDatabaseSearch = model.NewPointer(true)
-	}
```

Comment:
Thanks for this! Can we split it to another PR, thought? It feels like we may need to point to this change in the future, and it's quite hidden here.

Reply:
ok